templates/deployment_jobs.yaml

```diff
@@ -163,6 +163,15 @@ spec:
         {{- end }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+        {{- if and (regexMatch "^[0-9.]+$" .Values.image.tag) (semverCompare ">=2.110.0" .Values.image.tag) }}
```

Comment:
Do we ever release custom builds to customers, for one-off fixes? Idk if we do or not, but I was thinking of a scenario like a customer is on `2.115.0`, which would have this liveness probe enabled, and then we find some unrelated problem and give them a custom build tagged `2.115.0-finn-fix`, which would fail this regex match and get the liveness probe disabled. Thoughts?

Reply:
Good call out, I also don't know if we do this or not to be honest. I'm hoping that if we do such patches, we don't do it via `helm`. Otherwise, I'm not sure how to handle this in a non-hacky way. I guess I could strip all alphabetical characters and then strip any trailing dashes?

Reply:
Looks like this semverCompare can already deal with these...kinda. Helm just considers them "prerelease versions" and sorts them all as _less than_ the corresponding patch version. So maybe you could just remove the `$` end anchor from your regex and that's enough? Might also be something that #deployed-engineering could chime in on.

Reply:
thanks for investigating. removed the anchor!